### ISSUE TITLE
Error building this project

### ISSUE THREAD

**Issue by MartinLottering** [officialdoniald/Xamarin.Forms.Save.Open.PDF#1]
I'm cannot build this solution on Visual Studio 2019, version 16.5.5.

Severity	Code	Description	Project	File	Line	Suppression State
Error		Failed to create JavaTypeInfo for class: Android.Support.V4.View.Accessibility.AccessibilityManagerCompat/ITouchExplorationStateChangeListenerImplementor due to MAX_PATH: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Projects\Playground\Xamarin.Forms.Save.Open.PDF\Xamarin.Forms.Save.Open.PDF\Xamarin.Forms.Save.Open.PDF.Android\obj\Debug\81\android\src\mono\android\support\v4\view\accessibility\AccessibilityManagerCompat_TouchExplorationStateChangeListenerImplementor.java'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.File.InternalDelete(String path, Boolean checkHost)
   at System.IO.File.Delete(String path)
   at Xamarin.Android.Tools.Files.CopyIfStreamChanged(Stream stream, String destination)
   at Xamarin.Android.Tasks.Generator.CreateJavaSources(TaskLoggingHelper log, IEnumerable`1 javaTypes, String outputPath, String applicationJavaClass, String androidSdkPlatform, Boolean useSharedRuntime, Boolean generateOnCreateOverrides, Boolean hasExportReference)	Xamarin.Forms.Save.Open.PDF.Android

**Comment by MartinLottering** [comment]
Solved the Build Issue, I updated the nuget packages, and copied the solution to a shorter name directory.
Now it is building. Thanks.
